docs/develop-resources/deep-dives/3-add-mapper.md

```diff
@@ -51,4 +51,5 @@ If you are developing a Beta or more stable resource version, you should meet th
 
 * No `MISSING` comments left in the code
 * No `/*NOTYET*/` comments left in the code.
-* Each mapper method shall reflect in the `_http.log` as the value from `create.yaml` and `update.yaml` recorded in the `_http.log` POST and PUT/PATCH method.
\ No newline at end of file
+* A fuzzer test exists for both Spec and Status of the resource. [Example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/f313b00c52f09c4a52a2eb5fe2c15fa4b30a05fd/pkg/controller/direct/discoveryengine/fuzzers.go#L26-L47)
```

Comment:
It seem there is similar exit criteria in step "4-add-controller.md". Maybe step 3 is a better place.

Reply:
I think there's value in adding this earlier because we can probably catch issues in the controller ealier!

Reply:
( Also we can probably autogen this 💯 )